forge/__init__.py

```diff
@@ -29,20 +35,38 @@ def register_plugin(self, name, plugin, helptext):
         self.registry[name] = (plugin, helptext)
 
     def print_help(self):
+        """ Print Help For All Registered Plugins """
         help_entries = []
         for name in self.registry:
             help_entries.append([name, self.registry[name][1]])
         print(tabulate(help_entries, ['function', 'blurb']))
 
     def execute(self, command, args):
+        """ Execute Plugin """
         if command == 'help':
             self.print_help()
         else:
             self.registry[command][0](args)
 
+    def parse_conf(self, forge_dir, conf_file):
+        """ Parse Plugin Configuration File """
+        config = configparser.ConfigParser()
```

Comment:
Again, look at the os comment above.  This is not only non-portable, but fragile and a bad practice in python.

Reply:
just reading a file doesn't close it.  **with** is your friend here.  Resource leakage.

Reply:
config.read here is just calling the configparser api, I assume internally it handles resource management. When writing to it you'll notice we use `with`.